<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterStackDlg2</class>
 <widget class="QDialog" name="FilterStackDlg2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="AvailableFiltersList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="AddTopButton">
         <property name="text">
          <string>Add Top &gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddBottomButton">
         <property name="text">
          <string>Add Bottom&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RemoveButton">
         <property name="text">
          <string>&lt;&lt; Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="SelectedFilterList"/>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
